definitions/npm/react-i18next_v1.10.x/flow_v0.36.x-/react-i18next_v1.10.x.js

```diff
@@ -0,0 +1,35 @@
+import type { TFunction } from 'i18next'
```

Comment:
@ryyppy Updated to remove the type import from i18next. Apparently this was silently not working. Added a couple tests for it. Should be good to go unless you have any other suggestions.